src/sentry/static/sentry/app/components/dropdownAutoComplete/menu.tsx

```diff
@@ -0,0 +1,457 @@
+
+        // Items are loading if null
+        const itemsLoading = items === null;
+
+        // Has filtered results
+        const hasResults = !!autoCompleteResults.length;
+
+        // No items to display
+        const showNoItems = !busy && !filterValueOrInput && !hasItems;
+
+        // Results mean there was an attempt to search
+        const showNoResultsMessage =
+          !busy && !busyItemsStillVisible && filterValueOrInput && !hasResults;
+
+        // Hide the input when we have no items to filter, only if
+        // emptyHidesInput is set to true.
+        const showInput = !hideInput && (hasItems || !emptyHidesInput);
+
+        // When virtualization is turned on, we need to pass in the number of
+        // selecteable items for arrow-key limits
+        const itemCount =
+          virtualizedHeight && autoCompleteResults.filter(i => !i.groupLabel).length;
+
+        const renderedFooter =
+          typeof menuFooter === 'function' ? menuFooter({actions}) : menuFooter;
+
+        return (
+          <AutoCompleteRoot {...getRootProps()} className={rootClassName}>
+            {children({
+              getInputProps,
+              getActorProps,
+              actions,
+              isOpen,
+              selectedItem,
+            })}
+            {isOpen && (
+              <BubbleWithMinWidth
+                className={className}
+                {...getMenuProps({
+                  ...menuProps,
+                  style,
+                  css,
+                  itemCount,
+                  blendCorner,
+                  alignMenu,
+                  menuWithArrow,
+                })}
+              >
+                {itemsLoading && <LoadingIndicator mini />}
+                {showInput && (
+                  <InputWrapper>
+                    <StyledInput
+                      autoFocus
+                      placeholder={searchPlaceholder}
+                      {...getInputProps({...inputProps, onChange})}
+                    />
+                    <InputLoadingWrapper>
+                      {(busy || busyItemsStillVisible) && (
+                        <LoadingIndicator size={16} mini />
+                      )}
+                    </InputLoadingWrapper>
+                    {inputActions}
+                  </InputWrapper>
+                )}
+                <div>
+                  {menuHeader && <LabelWithPadding>{menuHeader}</LabelWithPadding>}
+                  <ItemList data-test-id="autocomplete-list" maxHeight={maxHeight}>
+                    {showNoItems && <EmptyMessage>{emptyMessage}</EmptyMessage>}
+                    {showNoResultsMessage && (
+                      <EmptyMessage>
+                        {noResultsMessage ?? `${emptyMessage} ${t('found')}`}
+                      </EmptyMessage>
+                    )}
+                    {busy && (
+                      <BusyMessage>
+                        <EmptyMessage>{t('Searching...')}</EmptyMessage>
```

Comment:
```suggestion
                        <EmptyMessage>{t('Searching\u2026')}</EmptyMessage>
```
Ellipses look better than three .